tests/monotouch-test/Metal/HeapDescriptorTest.cs

```diff
@@ -16,33 +16,76 @@ namespace MonoTouchFixtures.Metal {
 
 	[TestFixture]
 	public class HeapDescriptorTest {
+		MTLHeapDescriptor hd = null;
 
-		[Test]
-		public void Properties ()
+		[SetUp]
+		public void SetUp ()
 		{
 			TestRuntime.AssertXcodeVersion (8, 0);
 
+#if !MONOMAC
 			if (Runtime.Arch == Arch.SIMULATOR)
 				Assert.Ignore ("Type is missing on the simulator");
+#endif
+			
+			TestRuntime.AssertXcodeVersion (9, 0);
```

Comment:
Shouldn't this be inside an `#else ... #endif`? You're first asserting Xcode 8.0, then Xcode 9.0 otherwise, which looks strange.